qcengine/util.py

```diff
@@ -43,7 +43,7 @@ def create_mpi_invocation(executable: str, task_config: TaskConfig) -> List[str]
     mpirun_str = task_config.mpiexec_command.format(
         nnodes=task_config.nnodes,
         ranks_per_node=task_config.ncores // task_config.cores_per_rank,
-        total_ranks=task_config.nnodes * task_config.ncores,
+        total_ranks=task_config.nnodes * task_config.ncores // task_config.cores_per_rank,
```

Comment:
Can you add a test here in `test_config.py`?

Reply:
Yes. It turns out I already had a test for changing cores_per_rank, but it had the wrong number of total ranks. I've fixed the test to have the correct values.